server.py

```diff
@@ -649,6 +670,7 @@ def save_reports():
     results = {
         "status": "success",
     }
+    print(results)
```

Comment:
(same minor comment as above here re: print statement just to avoid adding new ones)